client/landing/subscriptions/components/tab-views/comments/comments.tsx

```diff
@@ -52,9 +52,10 @@ const Comments = () => {
 		<TabView errorMessage={ errorMessage } isLoading={ isLoading }>
 			<div className="comments-list-actions-bar">
 				<SearchInput
+					delaySearch
```

Comment:
In addition to providing `delaySearch` does it make sense to provide `delayTimeout={ 600 }` in order to maintain the same debounce timeout as before? The default one provided by `<Search />` is `300` right now:

https://github.com/Automattic/wp-calypso/blob/d2f640d00a8f4b4782dca1b9153454ed3fc30c43/client/components/search/index.jsx#L19

Reply:
I think it's better to standardize on the 300ms value. It's the `SearchInput` default, and also the other usage I'm fixing, the `useSearch` hook that this PR removes, uses the 300ms value.

In other words, let's standardize and unify the debounce timeouts, instead of preserving every random custom value.

Reply:
I don't have strong feelings about it, but 300ms did feel like a bit too low. When typing slowly it did some extra queries I didn't really need. Maybe that's just me and folks won't really perceive that.

Anyway, as long as this is an intentional decision, 300 ms sounds good to me.

Reply:
By the way, the `useDebouncedInput` in Gutenberg (`@wordpress/compose`) uses 250 ms.

Reply:
Do I detect a potential follow-up task to update the search delay to use `useDebouncedInput`? 😉

Reply:
I think 300ms is close enough to 250ms, it probably won't be worth doing 🙂